### PR TITLE
Lp-2519 Adding audio Xblock

### PR DESCRIPTION
**Pull request by javeriaejaz-arbisoft**

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2143/workflows/workflows_id/jobs/2143/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/constants.py (100%)
openedx/adg/lms/applications/helpers.py (100%)
openedx/adg/lms/applications/models.py (86.7%): Missing lines 106,109
openedx/adg/lms/applications/rules.py (100%)
openedx/adg/lms/applications/tests/constants.py (100%)
openedx/adg/lms/applications/tests/factories.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (100%)
openedx/adg/lms/applications/tests/test_helpers.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
openedx/adg/lms/courseware_override/tests/test_helpers.py (100%)
-------------
Total:   155 lines
Missing: 2 lines
Coverage: 98%
-------------
```

File: requirements/adg/base.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ mailchimp3==3.0.14
 pytest-mock==3.3.1
 mandrill==1.0.59
 -e git+https://${PHILU_PAT}@github.com/philanthropy-u/edx-msp-assessment-xblock.git@${ADG_XBLOCKS_BRANCH}#egg=msp-assessment-xblock
+git+https://github.com/edx/AudioXBlock.git@f668a4e231c5cdc338eef6bd148e57a02b48200b#egg=audio-xblock
```
Comment (asadali145) [inline requirements/adg/base.txt:5]: Why are we installing xblock from a specific commit?

Comment (javeriaejaz-arbisoft) [reply requirements/adg/base.txt:5]: If we do it from branch then any new change in the branch might conflict.

Comment (asadali145) [reply requirements/adg/base.txt:5]: How about fork it and keep it under our own organization?

Comment (asadali145) [reply requirements/adg/base.txt:5]: This way we won't have to worry about it.

Comment (javeriaejaz-arbisoft) [reply requirements/adg/base.txt:5]: Discussed with other. We only fork it when we have to customize the feature.

**Review by asadali145 — APPROVED** [review]
LGTM

**Review by harris-irfan — APPROVED** [review]
Great work with the debugging and research work 👍

**Pull request merged**